docs/language-server/protocol-language-server.md

```diff
@@ -490,26 +492,55 @@ interface SuggestionEntryAtom {
   /** The module name where the atom is defined. */
   module: string;
 
-  /** The list of arguments. */
-  arguments: SuggestionEntryArgument[];
+  /** The list of type parameters. */
+  params: SuggestionEntryArgument[];
 
   /** The type of an atom. */
   returnType: string;
```

Comment:
What is the returnType of the Type? I assume it will be always the type? I would left this field in Constructor, here it does not make much sense.

Reply:
Agree, it can be re-constructed from the module name and the type name. Removed